src/Symfony/Component/HttpKernel/Tests/Controller/ArgumentResolver/MapQueryParameterValueResolverTest.php

```diff
@@ -0,0 +1,343 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\Controller\ArgumentResolver;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Attribute\MapQueryParameter;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\MapQueryParameterValueResolver;
+use Symfony\Component\HttpKernel\Controller\ValueResolverInterface;
+use Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class MapQueryParameterValueResolverTest extends TestCase
+{
+    private ValueResolverInterface $resolver;
+
+    protected function setUp(): void
+    {
+        $this->resolver = new MapQueryParameterValueResolver();
+    }
+
+    /**
+     * @param array<mixed> $expected
+     *
+     * @dataProvider provideTestResolve
+     */
+    public function testResolve(
+        Request $request,
+        ArgumentMetadata $metadata,
+        array $expected,
+        string $exceptionClass = null,
+        string $exceptionMessage = null)
+    {
+        if ($exceptionMessage) {
+            self::expectException($exceptionClass);
+            self::expectExceptionMessageMatches($exceptionMessage);
+        }
+
+        self::assertSame($expected, $this->resolver->resolve($request, $metadata));
+    }
+
+    /**
+     * @return iterable<string, array{
+     *   Request,
+     *   ArgumentMetadata,
+     *   array<mixed>,
+     *   null|class-string<\Exception>,
+     *   null|string
+     * }>
+     */
+    public static function provideTestResolve(): iterable
+    {
+        yield 'parameter found and array' => [
+            Request::create('/', 'GET', ['ids' => ['1', '2']]),
+            new ArgumentMetadata(
```

Comment:
on one line + named arg to target the last (same below)

Reply:
Fixed.